src/coreclr/src/tools/Common/TypeSystem/Common/CastingHelper.cs

```diff
@@ -20,6 +20,79 @@ public static bool CanCastTo(this TypeDesc thisType, TypeDesc otherType)
             return thisType.CanCastToInternal(otherType, null);
         }
 
+        /// <summary>
+        /// Get TypeFlags of the reduced type of a type.
+        /// The reduced type concept is described in ECMA 335 chapter I.8.7
+        /// </summary>
+        static TypeFlags GetReducedTypeElementType(TypeDesc type)
```

Comment:
Nit: we follow the CoreFX coding guidelines that is always explicit about accessibility of methods/types.

```suggestion
        private static TypeFlags GetReducedTypeElementType(TypeDesc type)
```

Reply:
Right, it was just an overlook, thanks for spotting it.